posthog/tasks/exports/csv_exporter.py

```diff
@@ -71,9 +73,22 @@ def _export_to_csv(exported_asset: ExportedAsset, root_bucket: str) -> None:
 
 @app.task()
 def export_csv(exported_asset_id: int, root_bucket: str = settings.OBJECT_STORAGE_EXPORTS_FOLDER) -> None:
-    exported_asset = ExportedAsset.objects.get(pk=exported_asset_id)
-
-    if exported_asset.export_format == "text/csv":
-        return _export_to_csv(exported_asset, root_bucket)
-    else:
-        raise NotImplementedError(f"Export to format {exported_asset.export_format} is not supported")
+    timer = statsd.timer("csv_exporter").start()
+
+    exported_asset: Optional[ExportedAsset] = None
+    try:
+        exported_asset = ExportedAsset.objects.get(pk=exported_asset_id)
+        if exported_asset.export_format == "text/csv":
+            _export_to_csv(exported_asset, root_bucket)
+            statsd.incr("csv_exporter.succeeded", tags={"team_id": exported_asset.team.id})
+        else:
+            raise NotImplementedError(f"Export to format {exported_asset.export_format} is not supported")
```

Comment:
this is technically another failure we would like to plot?

Reply:
I left it out because it's only really logically possible... Maybe that makes it more important to assert has never happened 🤣